docs/security/secret-detection.md

```diff
@@ -64,7 +64,7 @@ If you would like to point the tool at a specific directory append the desired l
 
 If the pipeline is configured without the scanNonGitFiles is configured for your pipeline, then the `--all-files` argument will be set to scan git artifacts in your pipeline as well.
 
-`setFailureasWarning` will be set to false unless otherwise indicated, which will fail the task if a finding occurs.
```

Comment:
I think, looking at the context, that this should probably be setFailureAsWarning (but with lowercase a) - so this one was probably correct as it was

Reply:
thanks for noticing, fixed :)